docs/narr/hooks.rst

```diff
@@ -1660,18 +1665,19 @@ token unless ``disable_csrf=True`` is passed to the view:
    from pyramid.response import Response
    from pyramid.session import check_csrf_token
 
-   def require_csrf_view(view, info):
+   def csrf_view(view, info):
```

Comment:
This should probably match the actual `csrf_view` we ship more closely (related to the `val` stuff), and add just the check for `request.method == POST`.